Assets/Scripts/Core/AnimalEnums.cs

```diff
@@ -0,0 +1,39 @@
+namespace Anivision.Core
```

Comment:
Same namespace question here

Reply:
This file contains all of the animal enums that are used throughout the project so I thought it'd best to put it in core